### ISSUE TITLE
[Accessibilty] Colour contrast

### ISSUE THREAD

**Issue by davenichols-DHLS** [aemsites/danaher-abcam#171]
Sorry, I checked the pages, and they are failing the WCAG compliance, because of the green text colour that we are using (76, 166, 179).  Abcam seem to be using (55, 129, 137), which just passes.

Please can we amend the Stories pages to use the Abcam green colour.

https://pagespeed.web.dev/analysis/https-main--danaher-abcam--aemsites-aem-page-en-stories-articles-attitude-is-everything/x2u7mtxv9c?form_factor=mobile

**Comment by davenichols-DHLS** [comment]
FYI @Meenu-GSPANN  and @rgravitvl  - I created this.

**Comment by sunnyrah** [comment]
Hi @Rajesh-GSPANN , please replace:
1. the green color for 'Back' link text to #378189
2. the light grey color (#8B8B8B) for the 'resource type' card on the hub page to dark grey #435656  

cc: @Meenu-GSPANN  @davenichols-DHLS  @pardeepgera23

**Comment by Rajesh-GSPANN** [comment]
Hi @rgravitvl ,
Raised PR. Can you please have a look.
